File: users/admin/student/student.php
<?php
include_once "../../../connect.php";
class Student
{
  private $stud_number;
  private $first_name;
  private $last_name;
  private $id_nr;
  private $sex;
  private $phone;
  private $email;
  private $address;
  //member methods
  public function set_student($stud_number, $first_name, $id_nr, $sex, $phone, $email, $address)
  {
    return "Hi";
  }
  public function check_id($id_nr,$link)
  {
     $out= 1;
     $sql = "SELECT * FROM student WHERE id_nr  = \"$id_nr\" ";
     $result = mysqli_query($link, $sql);
     if(mysqli_num_rows($result) > 0)
     {
       $out = 0;
     }
     return $out;
  }
  public function validate_name($string)
  {
    //check it is not num
    //find the size of th string -
    //Trim white space
    $string = trim($string, " ");
    $out = 0;
    if(ctype_alpha($string))
    {
      $out = 1;
      return $out;
    }
    else
    {
      return $out;
    }
  }
  public function set_firstname($first_name)
  {
    $student = new Student();
    if($student->validate_name($first_name) ==1)
    {
      $this->first_name = $first_name;
    }
    else
    {
      $this->first_name = "NULL";
    }
  }
  public function get_firstname()
  {
    return $this->first_name;
  }
  public function set_surname($last_name)
  {
    $student = new Student();
    if($student->Validate_name($last_name) == 1)
    {
      $this->last_name = $last_name;
    }
    else
    {
      $this->last_name = "NULL";
    }
  }
  public function get_surname()
  {
    return $this->last_name;
  }
  public function validate_phone($phone,$link)
  {
     $out= 1;
     $sql = "SELECT * FROM student WHERE phone  = \"$phone\" ";
     $result = mysqli_query($link, $sql);
     if(mysqli_num_rows($result) > 0)
     {
       $out =0;
     }
     return $out;
  }
  public function validate_id($id_nr)
  {
    //Check the size
    $out = 0;
    $id_nr = trim($id_nr," ");
    if(ctype_digit($id_nr) && strlen($id_nr) ==13)
    {
      if(substr($id_nr,2,2) != "00")
      {
        if(substr($id_nr,4,2)!="00")
        {
          $out =1;
        }
      }
    }
    return $out;
  }
  public function set_id($id_nr)
  {
  $student = new Student();
  if($student->validate_id($id_nr) == 1)
  {
    $this->id_nr = $id_nr;
  }
  else
  {
   $this->id_nr = "NULL";
  }
  }
  public function get_id()
  {
  return $this->id_nr;
  }
  public function validate_sex($id_nr, $sex)
  {
    $out =0;
    $id_nr = substr($id_nr, 6, 4);
    if((int)$id_nr > 4999 && $sex =="M")
    {
      $out = 1;
      return $out;
    }
    elseif((int)$id_nr < 5000 && $sex == "F")
    {
      $out =1;
      return $out;
    }
    else
    {
      return $out;
    }
  }
}

File: database/student.cs
using System;
using System.Collections;
using System.Collections.Generic;
using System.Text;
namespace Grouping_system
{

    #region Student
    public class Student
    {
        #region Member Variables
        protected int _stud_number;
        protected string _first_name;
        protected string _last_name;
        protected string _gender;
        protected string _id_nr;
        protected string _phone;
        protected string _email;
        protected string _address;
        #endregion
        #region Constructors
        public Student() { }
        public Student(string first_name, string last_name, string gender, string id_nr, string phone, string email, string address)
        {
            this._first_name=first_name;
            this._last_name=last_name;
            this._gender=gender;
            this._id_nr=id_nr;
            this._phone=phone;
            this._email=email;
            this._address=address;
        }
        #endregion
        #region Public Properties
        public virtual int Stud_number
        {
            get {return _stud_number;}
            set {_stud_number=value;}
        }
        public virtual string First_name
        {
            get {return _first_name;}
            set {_first_name=value;}
        }
        public virtual string Last_name
        {
            get {return _last_name;}
            set {_last_name=value;}
        }
        public virtual string Gender
        {
            get {return _gender;}
            set {_gender=value;}
        }
        public virtual string Id_nr
        {
            get {return _id_nr;}
            set {_id_nr=value;}
        }
        public virtual string Phone
        {
            get {return _phone;}
            set {_phone=value;}
        }
        public virtual string Email
        {
            get {return _email;}
            set {_email=value;}
        }
        public virtual string Address
        {
            get {return _address;}
            set {_address=value;}
        }
        #endregion
    }
    #endregion
}